web_external/js/views/body/SubmissionView.js

```diff
@@ -64,6 +68,44 @@ covalic.views.SubmissionView = covalic.View.extend({
         }
     },
 
+    _progressHandler: function (progress) {
+        if (this.job.get('progress')) {
+            if (progress._id === this.job.get('progress').notificationId) {
+                if (progress.data.state === 'active') {
```

Comment:
Would it be more clear if this were a single boolean expression rather than three nested ifs?